### PR TITLE
Adds /item route

### PR DESCRIPTION
**Pull request by raq929**
Additionally, sets up a testing framework using pytest and pytest-flask-sqlalchemy

### CODE REVIEW

**Comment by leahvelleman** [comment]
Some general comments:

- Github CI says there are failing tests — we should either address that here or open a ticket to address it.
- README should probably address how to run python tests, even if it's obvious.

Also, see comments on individual files, and I may have more once I know how to run tests correctly

File: config.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from os import environ
+
+
+class Config:
+    # TESTING = environ.get('TESTING')
```
Comment (leahvelleman) [inline config.py]: What's the situation with these commented-out bits? Consider adding a comment capturing what they would do if uncommented and when/whether you think that will need to happen.

File: requirements.txt [hunk 2]
```diff
@@ -1,16 +1,25 @@
 alembic==1.4.2
+attrs==19.3.0
```
Comment (leahvelleman) [inline requirements.txt]: We can drop this requirement now — I was using it before I found out about built-in `@dataclass`, but it's no longer used anywhere in the PR code and I don't expect to use it in the future.

File: sabelotodo/app.py [hunk 3]
```diff
@@ -13,5 +15,11 @@
 def hello():
```
Comment (leahvelleman) [inline sabelotodo/app.py]: This code and `all_items()` below are duplicated here and in `routes.py`. Let's figure out which version is actually used/tested and drop the other one.

**Comment by raq929** [comment]
Run tests with` python -m pytest sabelotodo` from the root directory.
I'm working on getting the tests to pass on CI. They are passing locally for me.

File: README.md [hunk 4]
```diff
@@ -59,6 +60,19 @@ python manage.py db upgrade
 ```
 Rerun this last command after changing the models or checking out a new branch.
 
+For testing, a test database is set up. You will need to set database .env variables:
```
[the remark this answers is not in the document]
Comment (raq929) [reply README.md]: You do not have to create the test database yourself. You have to set these .env variables. The code will create the database. What would be a clearer way to say that?

File: config.py (hunk 1 above)
Comment (raq929) [reply config.py]: We're gonna fix it later. I should add a comment, but I'm lazy.

File: sabelotodo/app.py (hunk 3 above)
Comment (raq929) [reply sabelotodo/app.py]: Good catch. I think we can now drop this file entirely.

File: requirements.txt (hunk 2 above)
Comment (raq929) [reply requirements.txt]: It's required by another package, it turns out.

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (leahvelleman) [reply README.md]: Oh, I get it now. What about "To tell the test runner how to create it, add these lines to `.env`" or something like that? (Assuming I'm now understanding/describing it correctly...)

File: requirements.txt (hunk 2 above)
Comment (leahvelleman) [reply requirements.txt]: OHHHHH right ok. Good.

Comment (leahvelleman) [reply requirements.txt]: (This also answers my general question about `requirements.txt` above.)

Comment (raq929) [reply requirements.txt]: Yeah, we should do a review soonish, and uninstall the ones we don't know about one at a time and see if things break.

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (raq929) [reply README.md]: Not quite, but I'm updating and hopefully it will become clear. :)

File: config.py (hunk 1 above)
Comment (raq929) [reply config.py]: #32

**Comment by raq929** [comment]
#33 for fixing the github workflow.

**Pull request merged**